### ISSUE TITLE
UTMOS naturalness reported only for English/Chinese, not multilingual benchmarks — is naturalness measurement language-agnostic?

### ISSUE THREAD

**Issue by mukherjeesougata-eros** [k2-fsa/OmniVoice#217]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

In the paper, UTMOS is reported for the English and Chinese benchmarks only:

- Table 1 (LibriSpeech-PC test-clean, Seed-TTS test-en, Seed-TTS test-zh)
- Table 5, Table 7 (ablations, LibriSpeech-PC)
- Table 9 (inference-step comparison, English/Chinese)

However, the multilingual benchmarks report only intelligibility (WER/CER) and speaker similarity (SIM-o), with no naturalness metric (UTMOS):

- Table 3 (MiniMax-Multilingual-24)
- Table 4 (FLEURS-Multilingual-102)

May I know whether naturalness of speech is language-agnostic? Is that the reason naturalness was not reported on the multilingual benchmarks?

**Comment by zhu-han** [comment]
Several main reasons for not reporting UTMOS are: (1) Other papers also do not report UTMOS scores in massively multilingual benchmarks. We followed existing practices; (2) The UTMOS model is only trained on English, and their generalization to other languages may not be guaranteed. BTW, the UTMOS score itself is also reported to be unreliable in evaluating speech quality by some papers.

In terms of whether speech naturalness is language-agnostic, from the model architecture perspective, there are no language-specific designs, which means it's language-agnostic in a sense. But from the training data perspective, the quality and quantity of training data vary across different languages, and this will cause differences in the actual speech quality for each language.
